### PR TITLE
Feature/chef12

### PR DESCRIPTION
**Pull request by heathsnow**
- [ ] @laynes
- [x] @sweitzel74 

please review

### CODE REVIEW

File: rakefile [hunk 1]
```diff
@@ -1,11 +1,14 @@
-require 'tailor/rake_task'
+require 'rubocop/rake_task'
 require 'foodcritic'
 require 'rspec/core/rake_task'
 
-task :default => [:version, :tailor, :foodcritic, :spec]
+@provider = (ENV['PROVIDER'] || :virtualbox).to_sym
```
Comment (sneal) [inline rakefile]: This isn't used

**Comment by sneal** [comment]
:+1: for the TravisCI build

File: metadata.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
-name             'resharper'
-maintainer       'Daptiv Solutions LLC'
+name 'resharper'
+maintainer 'Daptiv Solutions LLC'
```
Comment (sweitzel74) [inline metadata.rb]: Changepoint

File: metadata.rb [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
 maintainer_email ''
```
Comment (sweitzel74) [inline metadata.rb]: this is needed

**Comment by sweitzel74** [comment]
LG other than the maintainer in the metadata, :shipit:  once those are updated.

**Pull request merged**